app/frontend/src/components/MarkdownInput.tsx

```diff
@@ -100,6 +102,11 @@ export default function MarkdownInput({
         "link",
       ],
     });
+
+    if (resetInputRef) {
+      resetInputRef.current = fieldRef.current.reset.bind(fieldRef.current);
+    }
```

Comment:
This is a bit of an annoying workaround, since react-hook-form's `fieldRef` through `controller` at the parent level doesn't seem to expose the same `fieldRef` as we set here - it only exposes an object with a `focus` method :/

Reply:
Horrific, but yeah doesn't seem like a way around it